<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simple</class>
 <widget class="QWidget" name="Simple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mult">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #FF9F0A;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(254, 194, 105);
	border: 20px;
	border-radius: 35px;
}

QPushButton:focus{
	color: #FF9F0A;
	background-color: rgb(255, 255, 255);
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>ร</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="div">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #FF9F0A;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(254, 194, 105);
	border: 20px;
	border-radius: 35px;
}

QPushButton:focus{
	color: #FF9F0A;
	background-color: rgb(255, 255, 255);
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>รท</string>
   </property>
  </widget>
  <widget class="QPushButton" name="point">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(0, 0, 0);
	background-color: #A5A5A5;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color:rgb(200, 200, 200);
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(0, 0, 0);
	background-color:rgb(230, 230, 230);
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{	
	text-align: right;
	font-family:&quot;Helvetica&quot;;
	font-size:44pt;
	border:none;
	color:white;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QPushButton" name="result">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #FF9F0A;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(254, 194, 105);
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255,255,255);
	background-color: rgb(254, 230, 194);
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mod">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(0, 0, 0);
	background-color: #A5A5A5;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color:rgb(200, 200, 200);
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(0, 0, 0);
	background-color:rgb(230, 230, 230);
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="x">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(0, 0, 0);
	background-color: #A5A5A5;
	border: 20px;
	border-radius: 35px;
	padding-bottom: 5px;
}

QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color:rgb(200, 200, 200);
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(0, 0, 0);
	background-color:rgb(230, 230, 230);
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>-x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plus">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;SF Mono Bold&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #FF9F0A;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(254, 194, 105);
	border: 20px;
	border-radius: 35px;
}

QPushButton:focus{
	color: #FF9F0A;
	background-color: rgb(255, 255, 255);
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minus">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	color: rgb(255, 255, 255);
	background-color: #FF9F0A;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(254, 194, 105);
	border: 20px;
	border-radius: 35px;
}

QPushButton:focus{
	color: #FF9F0A;
	background-color: rgb(255, 255, 255);
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-family: &quot;Helvetica&quot;;
	font-size: 20pt;
	text-align:left;
	padding-left: 25px;
	color: rgb(255, 255, 255);
	background-color: #333333;
	border: 20px;
	border-radius: 35px;
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #444444;
	border: 20px;
	border-radius: 35px;
}

QPushButton:pressed{
	color: rgb(255, 255, 255);
	background-color: #555555;
	border: 20px;
	border-radius: 35px;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
